setup.py

```diff
@@ -40,5 +40,5 @@
         'Programming Language :: Python :: 2',
         'License :: OSI Approved :: Apache Software License',
     ],
-    python_requires='<3',
+    python_requires='>=3',
```

Comment:
This is included in https://github.com/vivisect/vivisect/pull/357 It should be 3.6+ as vivisect master uses features introduced in Python 3.6 such as f-strings

Reply:
yup.  thanks!  i'm pulling this.